src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -260,7 +260,7 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
       ! It is only needed when redi mixing is enabled.
       if ( config_use_redi ) then
```

Comment:
Almost done... On this logical, we need to make sure that nothing in GM below depends on this Redi section. @vanroekel do you agree?

It looks like this section only computes `slopeTriadUp, slopeTriadDown, k33, k33Norm, RediKappaScaling` and those are not used in the sections below, so this if statement is correct.

Reply:
Confirmed with @vanroekel verbally, this is all correct.